### PR TITLE
Adding Followers' column on contributor's page

### LINKED ISSUE #5089: Add followers list to Contributors pages
We should show 2 columns on pages like https://publiclab.org/contributors/ui; one for people who have posted ("contributors" -- these are already shown on the page) and one for people who are Following the tag/topic. We currently only show followers here:

Screen Shot 2019-03-15 at 12 38 19 PM

Let's also rename the tab "People"! 

I'd love help with this - it could be done by making 2 columns, one labelled `People who've posted` using the current table -- code here:

https://github.com/publiclab/plots2/blob/master/app/views/tag/contributors.html.erb#L66

And adding a 2nd column displaying a similar table but for `Tag.followers(params[:id])` -- that is, followers of the topic. 

The tab text could be changed in this file: https://github.com/publiclab/plots2/blob/master/app/views/tag/show/_nav_tabs.html.erb

### PR DESCRIPTION
**Pull request by IshaGupta18**
Fixes #5089 

A column having a list of followers is added on contributor for tag page.

Also, the contributor's tab is renamed to `People`.

image

### CODE REVIEW

**Comment by IshaGupta18** [comment]
@jywarren kindly review my PR and do let me know what else can be done! Thanks a lot!

**Comment by IshaGupta18** [comment]
Thanks a lot! @gauravano what do you think about this?

File: app/views/tag/_contributors.html.erb [hunk 1]
```diff
@@ -40,19 +40,33 @@
...
-    </tr>
-    <% if @users %>
-      <% @users.each do |user| %>
-        <tr class="users-row">
-          <td><a href='/profile/<%= user.name %>'><i class="fa fa-user"></i> <%= user.name %></a></td>
-          <td><a href='/tag/<%= params[:id] %>/author/<%= user.name %>'><%= t('tag.contributors.notes') %> &raquo;</a></td>
+  <div class="col-md-6">
+    <table class="table">
+      <tr>
+        <th style="width:75%;"><%= t('tag.contributors.people_who\'ve_posted') %></th>
```
Comment (gauravano) [inline app/views/tag/_contributors.html.erb]: Hey @IshaGupta18, if you want to change the value, you can do https://github.com/publiclab/plots2/blob/master/config/locales/en.yml#L416 as this way, the translation value will be shown but after failing to find the key in en.yml.

Thanks!

Comment (IshaGupta18) [reply app/views/tag/_contributors.html.erb]: So I have done this now. Anything else that can be done @gauravano ?

**Comment by IshaGupta18** [comment]
@jywarren what do you think of this?

**Comment by IshaGupta18** [comment]
@gauravano do you think this is goos to go now?

File: app/views/tag/show/_nav_tabs.html.erb [hunk 2]
```diff
@@ -12,7 +12,7 @@
     again, can we have a generic list of contributors that is
     calculated in the controller? -->
     <% if !@wildcard %>
-      <li<% if @node_type == "contributors" %> class="active"<% end %>><a href="/contributors/<%= params[:id] %>"><i class="fa fa-user"></i> <span class="hidden-sm hidden-xs"><%= raw t('tag.show.contributors') %></span><span class="badge"><%=@length %></span></a></li>
+      <li<% if @node_type == "contributors" %> class="active"<% end %>><a href="/contributors/<%= params[:id] %>"><i class="fa fa-user"></i> <span class="hidden-sm hidden-xs"><%= raw t('tag.show.people') %></span> <span class="badge"><%=@length %></span></a></li>
```
Comment (gauravano) [inline app/views/tag/show/_nav_tabs.html.erb]: @IshaGupta18 please add value corresponding to tag.show.people

Comment (gauravano) [reply app/views/tag/show/_nav_tabs.html.erb]: Regarding names, seems good to me

**Comment by IshaGupta18** [comment]
Okay! I will add this! Thanks a ton!

**Comment by IshaGupta18** [comment]
@gauravano done!

**Comment by IshaGupta18** [comment]
@gauravano is it good to go now?

**Comment by IshaGupta18** [comment]
@jywarren could you please review this PR? Thanks a lot!

**Comment by IshaGupta18** [comment]
@SidharthBansal could you please give a review on this PR?

File: config/locales/en.yml [hunk 3]
```diff
@@ -386,7 +386,7 @@ en:
       maps: "Maps"
       no_results_found: "No results found; try searching for '<b>%{tag}</b>'"
       try_advanced_search: "Or try an <a href='%{url1}'>advanced search</a>"
-      contributors: "Contributors"
```
Comment (gauravano) [inline config/locales/en.yml]: Actually, keep the contributors too as we are using tag.show.contributors in line 40

Comment (IshaGupta18) [reply config/locales/en.yml]: Done!

**Comment by IshaGupta18** [comment]
@gauravano what else can I do here?

File: config/locales/en.yml [hunk 4]
```diff
@@ -386,6 +386,7 @@ en:
       maps: "Maps"
       no_results_found: "No results found; try searching for '<b>%{tag}</b>'"
       try_advanced_search: "Or try an <a href='%{url1}'>advanced search</a>"
+      people: "Contributors"
```
Comment (gauravano) [inline config/locales/en.yml]:
```suggestion
      people: "People"
```

Comment (IshaGupta18) [reply config/locales/en.yml]: Sincerest apologies!

Comment (gauravano) [reply config/locales/en.yml]: no issue!

Comment (IshaGupta18) [reply config/locales/en.yml]: So, is it good to go now?

Comment (gauravano) [reply config/locales/en.yml]: yes :smile:

**Comment by gauravano** [comment]
Hi @jywarren, please merge it after merging the bootstrap4 PR as this PR touches views and so might conflict the Bootstrap4 PR which is scheduled to be merged today. Thanks!

**Comment by gauravano** [comment]
Hi @IshaGupta18, please remove the merge conflict. Thanks!

**Comment by IshaGupta18** [comment]
Done!

**Comment by gauravano** [comment]
the name of tab is "contributors" after you resolved the conflict

**Comment by gauravano** [comment]
Also, there's some UI issues. Here's updated view:

Screenshot from 2019-04-25 23-19-57

UI changes are due to bootstrap4 upgrade probably but we have to resolve them too, so sorry for the inconvenience.

**Comment by IshaGupta18** [comment]
Really not a problem, I'll pull in the fresh changes and update by PR accordingly. Is that cool?

**Comment by gauravano** [comment]
Yes, thank you!

**Comment by IshaGupta18** [comment]
@gauravano I have created a new PR #5632 with the updates, kindly review that one and I am closing this. Thanks a lot!

**Comment by gauravano** [comment]
ok. Great!

**Pull request closed without merging**